libpod/container_internal_common.go

```diff
@@ -2306,8 +2308,13 @@ func (c *Container) addHosts() error {
 	}
 
 	var exclude []net.IP
+	var hostContainersInernalIP string
```

Comment:
I'm guessing you meant `Internal` instead of `Inernal`?

Reply:
yes